lib/index.ts

```diff
@@ -22,63 +20,65 @@ import template from 'lodash.template';
 import plur from 'plur';
 import unusedFilename from 'unused-filename';
 
-type PageresStream = Readable & {filename: string};
+interface PageresStream extends Readable {
+	filename: string;
+}
```

Comment:
What's the benefit of using `interface` here? The type existing type annotation works in TS too.